### PR TITLE
Add ipmi-sim-console into Infrasim App

### PR DESCRIPTION
**Pull request by sharkconi**
Signed-off-by: sharkconi <sharkconi@gmail.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
@xiar @fub2 @PaynePei @JuneZhou19 @turtle-fly

File: bin/infrasim-console [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+'''
```
Comment (fub2) [inline bin/infrasim-console]: @helloarys is moving all this kind of staff into repo of ipmi-sim-console.

Comment (sharkconi) [inline bin/infrasim-console]: @fub2 I know what arys does.  But I prefer to move them into infrasim repo, so they seems to be  a part of infrasim application

**Comment by sharkconi** [comment]
+1

**Pull request merged**